Source/Build Specs/ni_protobuf_types.vipb

```diff
@@ -183,5 +180,76 @@
     </VI_Docs>
     <System_Destinations/>
   </Advanced_Settings>
-  <Library_Palette_Definition/>
+  <Library_Palette_Definition>
```

Comment:
Did you add this palette? I'm not opposed to adding this to the palette, but it is not in a state right now where it is shippable. We would at least need icons. As long as there is a follow-up user story to make the palette shippable then I am okay with this.

Reply:
@bkeryan I can take this over and push it through if you like since you're out this week.